<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janelaaddnovafaixa</class>
 <widget class="QDialog" name="janelaaddnovafaixa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Nova Faixa</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconspotper.qrc">
    <normaloff>:/new/prefix1/imgs/spot.png</normaloff>:/new/prefix1/imgs/spot.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	font: 75 16pt &quot;Bahnschrift&quot;;
	background-color:#44c767;
	border-radius: 10px;
	border:1px solid #18ab29;
	color:#ffffff;
	font-size:17px;
}
QPushButton:hover {
	background-color: #53f37e;
}
QDialog{
	background-color: #303030;
}
QLabel{
	font: 75 16pt &quot;Bahnschrift&quot;;
	color:#ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Albuns&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetAlbuns"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnPesquisarFaixas">
       <property name="text">
        <string>Pesquisar Faixas</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Faixas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetFaixas"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddAPlaylist">
       <property name="text">
        <string>Adicionar a Playlist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iconspotper.qrc"/>
 </resources>
 <connections/>
</ui>
